boards/arm/atsamr21_xpro/atsamr21_xpro_defconfig

```diff
@@ -1,3 +1,4 @@
+# Copyright (c) 2019 Gerson Fernando Budke
```

Comment:
I'd personally find it strange that you include copyright for addition of 2 config options, while don't do anything re: atsamr21_xpro/pinmux.c . Clearly, there's more effort went into it, whereas adding 2 config options is ~ trivial?

Reply:
The #21353 that add this copyright was already accepted before your review. It enables #20296 that was the real contribution. I'll be more careful in future and if this is a problem I can send an update to drop it.

Reply:
Not a problem requiring revert from my PoV. Thanks for reply.